Tests/ClientTests/Coordinators/RouteTests.swift

```diff
@@ -0,0 +1,215 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/
+
+import XCTest
+@testable import Client
+
+class RouteTests: XCTestCase {
+    var routeBuilder: RouteBuilder!
+
+    override func setUp() {
+        super.setUp()
+        self.routeBuilder = RouteBuilder { false }
```

Comment:
If we setup a tests variable, we should also set it to nil. You can change that in the next PR if that's the only change needed (let's see if Orla finds something else). 
```
    override func tearDown() {
        super.tearDown()
        routeBuilder = nil
    }
```